app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/ApplicationController.scala

```diff
@@ -33,26 +34,33 @@ import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.{ExecutionContext, Future}
 
+
+// TODO Get rid of config, env and authConnector
+// TODO Get rid of AuthorisedFunctions and AuthRedirectSupport trait
```

Comment:
You still planning on doing these TODOs?

Reply:
Yeah, but the test fixes are huge, so I left this for now.

It's better than injecting application, but make a duplication, because all those information are in the AuthAction

Reply:
Will cover this in a separate PR